include/dca/linalg/blas/kernels_gpu.hpp

```diff
@@ -32,6 +32,9 @@ inline void copyRows(int row_size, int n_rows, const int* i_x, const std::comple
   auto cu_y = util::castCudaComplex(y);
   copyRows(row_size, n_rows, i_x, cu_x, ldx, i_y, cu_y, ldy, thread_id, stream_id);
 }
+template <typename Type>
```

Comment:
How does this differ from the copyRows method above.  Unexplained overloads frequently become cryptic over time.

Reply:
This file is an implementation of matrixop.hpp. I mentioned it in the file description, as all the docs are in the header.